lib/shutterstock-ruby/videos.rb

```diff
@@ -10,9 +10,9 @@ def details(id, options = {})
       JSON.parse(get('/videos', { id: id }.merge(options)))
     end
 
-    def purchase(id, subscription_id, size, options = {})
+    def purchase(id, subscription_id, size, options = {}, metadata = {})
```

Comment:
I would have liked to have `metadata` before `options` but that would be backwards incompatible

Reply:
Would it make sense for `metadata` to be an entry in `options`?

Reply:
Yeah it could, but we don't want it to be passed to `post` as an option

Reply:
Oh yeah, I see.